atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -221,6 +221,18 @@ def set_operators_metadata(self, extra, worker_metadatas):
 
                     return  # only one worker can process operator manifests
 
+    def set_pnc_build_metadata(self, extra):
+        plugin_results = self.workflow.postbuild_results.get(PLUGIN_GENERATE_MAVEN_METADATA_KEY) \
+                         or {}
+        pnc_build_ids = plugin_results.get('pnc_build_ids')
+        pnc_build_metadata = {'pnc': {}}
+
+        if pnc_build_ids:
+            pnc_build_metadata['pnc']['build_ids'] = pnc_build_ids
+
+        if pnc_build_metadata['pnc']:
+            extra['image']['pnc'] = pnc_build_metadata['pnc']
+
```

Comment:
why not just :

if pnc_builds_ids:
   pnc = {'build_ids': pnc_build_ids}
   extra['image']['pnc'] = pnc